src/client.ts

```diff
@@ -306,7 +306,8 @@ export class SubscriptionClient {
           if (parsedMessage.payload.data && !parsedMessage.payload.errors) {
```

Comment:
Probably cleaner to conditionally set `playloadData` and `playloadErrors` and have only a single call to `this.subscriptions[subId].handler`.